src/constants/src/AbsolutePosition.js

```diff
@@ -1,18 +1,18 @@
 export default {
   TOP_LEFT: {
-    top: 0,
-    left: 0
+    top: 50,
+    left: 50
   },
   TOP_RIGHT: {
-    top: 0,
-    right: 0
+    top: 50,
+    right: 50
   },
   BOTTOM_LEFT: {
-    bottom: 0,
-    left: 0
+    bottom: 50,
+    left: 50
   },
   BOTTOM_RIGHT: {
```

Comment:
We probably need to define these using the `Position` constants that Evergreen exports so they are always consistent.

e.g. `[Position.BOTTOM_RIGHT]: {`